test/e2e/testkit/matchers/metric_matchers.go

```diff
@@ -34,6 +34,36 @@ func HaveMetrics(expectedMetrics ...pmetric.Metric) types.GomegaMatcher {
 	}, gomega.ContainElements(expectedMetrics))
 }
 
+func HaveSumMetrics(expectedMetrics ...pmetric.Metric) types.GomegaMatcher {
```

Comment:
Add unittests for the. Take a look how the other matchers are tested in metric_matchers_test.go.